systemtests/tests/backup-bareos-test/etc/bareos/bareos-sd.d/device/FileStorage.conf.in

```diff
@@ -1,7 +1,7 @@
 Device {
   Name = FileStorage
   Media Type = File
-  Archive Device = @archivedir@
+  Archive Device = storage
```

Comment:
it this change is intentional, you can also make the conf.in into a .conf file, because there are no placeholders left that could be replaced.
However, if it needs to be changed here, it probably needs to be changed in all other tests, too.